### DISCUSSION TITLE
希望加个登录页

### DISCUSSION THREAD

**Discussion by ghost** [ChatGPTNextWeb/NextChat#843] [Q&A]
可以环境变量里添加用户名和密码，以实现能保存用户的历史记录，以及保存用户的设置

**Comment by cyj153** [comment]
历史记录，用户设置都是保存在本地的吧，就算能登录也没办法保留

**Comment by henryhu712** [comment]
登录涉及用户管理，更多涉及的是后端用户权限管理。这个项目恐怕主要是前端交互，不是全栈应用。我自己做了一个授权登录的，后端用nest.js，网址 https://backre.com/

**Reply by kaynai** [reply]
这个网站源码请问可以分享吗 是需要授权吗
